src/PublicSuffixListManager.php

```diff
@@ -35,6 +35,15 @@ class PublicSuffixListManager
     const PRIVATE_PSL_PHP_FILE = 'private-public-suffix-list.php';
 
     /**
+     * @var PublicSuffixList Public Suffix List
+     */
+    protected static $domainList = [
+        self::ALL_DOMAINS => self::PDP_PSL_PHP_FILE,
+        self::ICANN_DOMAINS => self::ICANN_PSL_PHP_FILE,
+        self::PRIVATE_DOMAINS => self::PRIVATE_PSL_PHP_FILE,
+    ];
+
+    /**
      * @var string Public Suffix List URL
      */
     protected $publicSuffixListUrl = 'https://publicsuffix.org/list/effective_tld_names.dat';
```

Comment:
We should update this to `https://publicsuffix.org/list/public_suffix_list.dat`

Reply:
updated